features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/EnrichedResource.kt

```diff
@@ -32,7 +32,9 @@ internal data class EnrichedResource(
     }
 
     internal companion object {
-        internal const val APPLICATION_ID_KEY = "application_id"
+        internal const val APPLICATION_ID_RESOURCE_KEY = "applicationId"
+        internal const val APPLICATION_ID_OUTER_KEY = "application"
+        internal const val APPLICATION_ID_INTERNAL_KEY = "id"
```

Comment:
maybe we need some explanation what outer vs internal key means, not quite clear

Reply:
The api is for a nested object -> something like { application: { id: value }}. I can add a comment there, or perhaps rename to `APPLICATION_KEY` and `ID_KEY`?

Reply:
yes, this makes much more sense - these are just string literal which are supposed to be used as keys, no matter where.